src/core/scene/a-scene.js

```diff
@@ -464,7 +464,7 @@ module.exports.AScene = registerElement('a-scene', {
         // VR mode. The canvas is resized by VREffect following the values returned
```

Comment:
Can we update the comment to reflect the new logic? Something like this:
```Do not resize canvas If the a camera or canvas have not been injected. On VR mode, VREffect handles canvas resize based on the dimensions returned by the getEyeParameters function of the WebVR API. We want to honor those values instead of overwriting the size with the windows and width and height```

Reply:
Updated, is this clear enough?